example/90-shoot-aws.yaml

```diff
@@ -30,6 +30,7 @@ spec:
       zones: ['eu-west-1a']
   kubernetes:
     version: 1.12.1
+    # allowPrivilegedContainers: false
```

Comment:
Could you add a comment here to explain what that does if set to true/false? (from [here](https://github.com/gardener/gardener/pull/226/files#r228191775))